packages/sdk/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@voltz-protocol/sdk-v2",
+  "version": "1.33.0",
+  "description": "A TypeScript wrapper for the Voltz V2 smart contract",
+  "main": "dist/index.js",
+  "types": "dist/types",
+  "publishConfig": {
+    "access": "public"
+  },
+  "license": "MIT",
+  "scripts": {
+    "prebuild": "yarn code-quality:check && rm -rf dist",
+    "build": "yarn prebuild && tsc -p tsconfig.build.json",
+    "start": "npx ts-node src/index.ts",
+    "ts:check-types": "npx tsc --noEmit",
+    "ts:circular-check": "dpdm --no-output --no-tree --no-warning --exit-code circular:1 -T ./src/index.ts",
+    "ts:check": "yarn ts:check-types && yarn ts:circular-check",
+    "lint:check": "eslint '{src,tests}/**/*.ts'",
+    "prettier:check": "npx prettier --check './{src,tests}/**/*.ts'",
+    "code-quality:check": "npx concurrently 'yarn ts:check' 'yarn prettier:check' 'yarn lint:check --quiet'",
+    "lint:fix": "npx eslint --fix '{src,tests}/**/*.ts'",
+    "prettier:fix": "npx prettier --write './{src,tests}/**/*.ts'",
+    "test": "mocha --require ts-node/register tests/**/*.test.ts",
+    "test:coverage": "yarn test --coverage",
+    "generate:coverage-badges": "npx istanbul-badges-readme --silent",
+    "release": "semantic-release --node-version=18 --extends=../../release.config.js --require=ts-node/register"
+  },
+  "packageManager": "yarn@3.5.1",
+  "devDependencies": {
+    "@commitlint/cli": "^13.1.0",
+    "@commitlint/config-conventional": "^13.1.0",
+    "@jest/globals": "^29.3.1",
+    "@semantic-release/git": "^10.0.0",
+    "@semantic-release/npm": "^8.0.2",
+    "@sentry/cli": "2.10.0",
+    "@types/big.js": "^6.1.3",
+    "@types/chai": "^4.3.5",
+    "@types/crypto-js": "^4.1.1",
+    "@types/faker": "^5.5.8",
+    "@types/jest": "^29.2.4",
+    "@types/lodash": "^4.14.174",
+    "@types/luxon": "^2.0.9",
+    "@types/mocha": "^9.1.1",
+    "@types/sinon": "^10.0.14",
+    "@typescript-eslint/eslint-plugin": "4.31.2",
+    "@typescript-eslint/parser": "4.31.2",
+    "chai": "^4.3.7",
+    "commitizen": "^4.2.4",
+    "concurrently": "7.6.0",
+    "eslint": "7.32.0",
+    "eslint-config-prettier": "8.3.0",
+    "eslint-plugin-import": "2.26.0",
+    "eslint-plugin-prettier": "4.0.0",
+    "eslint-plugin-unused-imports": "2.0.0",
+    "istanbul-badges-readme": "1.8.4",
+    "jest": "^29.3.1",
+    "lint-staged": "11.1.2",
+    "mocha": "^10.2.0",
+    "prettier": "2.4.1",
+    "sinon": "^15.0.4",
+    "ts-jest": "^29.0.3",
+    "ts-mockito": "^2.6.1",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.4.3",
+    "yalc": "^1.0.0-pre.53"
+  },
+  "dependencies": {
+    "@metamask/detect-provider": "^1.2.0",
+    "@sentry/browser": "7.25.0",
+    "@sentry/tracing": "7.25.0",
+    "@uniswap/sdk-core": "3.0.1",
```

Comment:
Can we get rid of uniswap dependencies? It seems that we use v. small utilities from their codebase. It'd be better to bring them here for better control.

Reply:
hmmm I initially tried to copy the code we need but it has a lot of dependencies and it was very difficult to follow. We can definitely rewrite these dependencies, get ride of JSBI and all the stuff we don't need. I'll try looking into it for the next PR